sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/serializer/JsonSerializer.java

```diff
@@ -11,29 +12,7 @@
 /**
  * Generic interface covering basic JSON serialization and deserialization methods.
  */
-public interface JsonSerializer extends ObjectSerializer {
-    /**
-     * Reads a JSON stream into its object representation.
-     *
-     * @param stream JSON stream.
-     * @param typeReference {@link TypeReference} representing the object.
-     * @param <T> Type of the object.
-     * @return The object represented by the deserialized JSON stream.
-     */
-    @Override
-    <T> T deserialize(InputStream stream, TypeReference<T> typeReference);
-
-    /**
-     * Reads a JSON stream into its object representation.
-     *
-     * @param stream JSON stream.
-     * @param typeReference {@link TypeReference} representing the object.
-     * @param <T> Type of the object.
-     * @return The object represented by the deserialized JSON stream.
-     */
-    @Override
-    <T> Mono<T> deserializeAsync(InputStream stream, TypeReference<T> typeReference);
-
+public interface JsonSerializer extends com.azure.core.util.serializer.JsonSerializer {
```

Comment:
Why it is still needed in core-experiemental?

Reply:
There is still prototyped work for JsonSerializer that cannot move into Azure Core, yet/ever (may never do it). I could rename the JsonSerializer in Azure Core Experimental to JsonSerializerExtensions to make it less confusing.